### ISSUE TITLE
Some elements are missing the focus indicator on iPhone 7

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#6665]
These elements are missing the focus indicator on iPhone 7:

- [ ] mobile menu links
- [ ] forgot password link
- [ ] footer links
- [ ] checkboxes
- [ ] radio buttons
- [ ] buttons
